time init_t
Count count
Matrix m